awxkit/test/cli/test_authentication.py

```
@@ -44,6 +44,53 @@ def setup_session_auth(cli_args: Optional[List[str]] = None) -> Tuple[CLI, Mock,
+        awx login --conf.host URL --conf.username USER --conf.password PASS
+        # Returns: {"token": "E*******J"}
+
+        awx --conf.host URL --conf.token E*******J job_templates launch ...
+        # This WORKED in 4.6.12
+
+    BREAKING CHANGE: Version 4.6.21 removed token authentication entirely,
+    causing customer to report: "neither token no username/password are working"
+
+    This test will FAIL with current code and PASS once fixed.
+    """
+    cli, mock_root, mock_connection = setup_token_auth(['awx', '--conf.host', 'https://aap-sbx.cambiahealth.com', '--conf.token', 'E1234567890J'])
```

Ruff (0.15.0)

[warning] Unpacked variable `mock_root` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)